engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -136,6 +146,35 @@ private void loadModulesFromApplicationPath(PathManager pathManager) {
         scanner.scan(registry, paths);
     }
 
+    private void loadModulesFromClassPath() {
+        ClasspathCompromisingModuleFactory moduleFactory = (ClasspathCompromisingModuleFactory) this.moduleFactory;
+        for (String metadata_name : moduleFactory.getModuleMetadataLoaderMap().keySet()) {
```

Comment:
We usually don't use snake case in our Java code. 
```suggestion
        for (String metadataName : moduleFactory.getModuleMetadataLoaderMap().keySet()) {
```